packages/core/src/ThemeProvider.js

```diff
@@ -1,15 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import styled, {
-  ThemeProvider as StyledThemeProvider,
-  injectGlobal
-} from 'styled-components'
+import styled, { ThemeProvider as StyledThemeProvider } from 'styled-components'
 import nextTheme from './theme'
 
-injectGlobal`body {
-  margin: 0;
-}`
-
```

Comment:
Is the removal of `margin: 0` on global injected style agreed by WG?

My hunch is the 0 margin on the global of the injected storybook style would omit any ambiguous margins outside of the component being developed. So perhaps we can confirm this removal of the injected global style w. 0 margin?

Reply:
we talked about it in the last meeting, most people seemed fine with it